package.json

```diff
@@ -2,38 +2,37 @@
   "name": "kontist",
   "version": "0.19.0",
   "description": "Kontist client SDK",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/lib/index.js",
+  "types": "dist/lib/index.d.ts",
   "repository": "https://github.com/kontist/js-sdk.git",
   "author": "Kontist GmbH",
   "license": "Apache-2.0",
   "scripts": {
     "build": "tsc && npx webpack",
-    "test": "mocha -r ts-node/register --recursive 'tests/**/*.spec.ts'",
+    "test": "mocha --recursive 'dist/tests/**/*.spec.js'",
```

Comment:
This works fine but only if we `npm run build`  before running the tests 🤔if `dist` is not present it will error, and if for example I am changing my tests while doing a feature I would need to manually `npm run build` before `npm t`.

Do you think we should do `"test": "npm run build && mocha --recursive 'dist/tests/**/*.spec.js'"` or something similar?

I might be missing something though 😄

Reply:
Good catch, thank you. Fixed in d4e6578a21a91f2ee53be14438574f35e1d596d4